nixnet/database/_collection.py

```diff
@@ -47,30 +48,57 @@ def __len__(self):
     def __iter__(self):
         return self.keys()
 
-    def __getitem__(self, index):
-        if isinstance(index, six.string_types):
-            ref = _funcs.nxdb_find_object(self._handle, self._type, index)
+    def __getitem__(self, key):
+        """Return the database object.
+
+        Args:
+            Name of database object
+        Returns:
+            Database object
+        """
+        if isinstance(key, six.string_types):
+            ref = _funcs.nxdb_find_object(self._handle, self._type, key)
             return self._factory(ref)
         else:
-            raise TypeError(index)
+            raise TypeError(key)
```

Comment:
I'll revert, but the only questions I have is whether `close_all_refs` makes sense in the python API, since we don't refer to refs otherwise. Does @epage have any thoughts on this?

Reply:
Just realized I should add docs for Database ``__init__`` and that the C API docs on nxdbOpenDatabase explain refs in context of opening/closing a database. ``close_all_refs`` makes sense.